2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -8,5 +8,17 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
    `list-item`.
 3. Look in the css file!
 ------------------------------------------------------------------------------*/
+const nicknameEl = document.getElementById("nickname");
+nicknameEl.textContent = "Crazzy Bob";
 
-// TODO add your JavaScript code here.
+const favFoodEl = document.getElementById("fav-food");
+favFoodEl.textContent = "Pizza";
+
+const hometownEl = document.getElementById("hometown");
+hometownEl.textContent = "New York";
+
+const newClass = document.querySelectorAll("li");
+
+for (let i in newClass){
```

Comment:
Try not to use `for in`, some weird stuff can happen!

```suggestion
newClass.forEach((li) => {
```